upi/openstack/network.yaml

```diff
@@ -114,6 +126,40 @@
       cmd: "openstack router set --tag {{ cluster_id_tag }} {{ os_router }}"
     when: os_external_network is defined and os_external_network|length>0
 
+  - name: 'Get information about router'
+    ansible.builtin.command:
+      cmd: "openstack router list --name {{ subnet_pool }} --tags {{ cluster_id_tag }}"
+    register: router
+    when:
+    - os_subnet6_range is defined
+    - "{{ os_subnet6_range|ansible.utils.ipv6 }}"
+
+  - name: 'Create external router IPv6'
```

Comment:
we can check if the external_network is defined like[ is done here](https://github.com/openshift/installer/pull/7633/files#diff-810ef9226dc20e51f9b2814e9885d243be1590bff38badb8d2b341263915d30cR122) plus check the ipv6 cidr is defined and just add a task to add the ipv6 interface to the router. I believe we can use the router name instead of checking if the router exists. I believe this will save some steps.

I don't see a need to create a separate router given we already have one.

Reply:
Yeah, I think I overdid it :sweat_smile: